appsdk/sdk.go

```diff
@@ -364,6 +365,7 @@ func (sdk *AppFunctionsSDK) Initialize() error {
 			handlers.NewClients().BootstrapHandler,
 			handlers.NewTelemetry().BootstrapHandler,
 			handlers.NewVersionValidator(sdk.skipVersionCheck, internal.SDKVersion).BootstrapHandler,
+			message.NewBootstrap(clients.CoreCommandServiceKey, internal.ApplicationVersion).BootstrapHandler,
```

Comment:
Change `clients.CoreCommandServiceKey` to `sdk.ServiceKey`. Sorry, missed this in your question earlier.

Reply:
Thanks for the feedback. Fixed in a69513d